src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -47,6 +53,16 @@ private static void setupCommandTests(Application application) {
 
 		application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));
 
+		application.addTest("Visitor test DriverCommand", event -> {
+			DriverCommand command = new OperateToCommand(0, 0);
+			Visitor visitor = new DriverCommandVisitor();
+			((OperateToCommand) command).accept(visitor);
```

Comment:
With visitor pattern casting shouldn't be necessary. Fixing another issues may help.